studio/components/ui/PageTelemetry.tsx

```diff
@@ -56,6 +59,8 @@ const PageTelemetry: FC = ({ children }) => {
       })
 
       post(`${API_URL}/telemetry/pageview`, {
+        ...(ref && { projectRef: ref }),
+        ...(selectedOrganization && { orgSlug: selectedOrganization.slug }),
```

Comment:
sanity check this please 
@alaister

Reply:
i guess its just do we need the org slug if the page view is on a project?
`selectedOrganization` is fine i think, but that will incur a request to the organizations endpoint if its not loaded in RQ yet

otherwise { slug } from useParams might be sufficient

Reply:
Preferably we don't want to incur extra requests on the front end right? Does it make our site slower? Having org slug if the pageview is on a project is a nice to have.

Reply:
Yep, this looks good! `selectedOrganization` will most likely be cached so this shouldn't slow things down at all :)